virttest/libvirt_xml/vm_xml.py

```diff
@@ -557,18 +557,21 @@ def __init__(self, hypervisor_type='kvm', virsh_instance=base.virsh):
         self.xml = u"<domain type='%s'></domain>" % hypervisor_type
 
     @staticmethod  # static method (no self) needed b/c calls VMXML.__new__
-    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh):
+    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh,
```

Comment:
Here we have virsh_instance, uri is just part of it, so there's no need to add this param